docs/menus.md

```diff
@@ -59,8 +59,21 @@ getMenuItems() {
   return [{
     id         : 'hello-world',
     label      : 'Hello World',
+    label      : (editor, block, menuItem) => `My special block ${block.id}`,
     onClick    : (editor, block, menuItem) => alert(`My block is ${ block.id }`),
     isDisabled : (editor, block, menuItem) => false
   }]
 }
 ```
+
+Another example:
+
+```javascript
+getMenuItems() {
+  return [{
+    id         : 'hello-world',
+    label      : () => `Toggle position: ${this.positionLabel()}`
+    onClick    : () => this.togglePosition()
```

Comment:
These values feel a little far away to me. What do you think about aligning these `:` closer to the `onClick`?